### DISCUSSION TITLE
Alternate CTs?

### DISCUSSION THREAD

**Discussion by ADKMechETECH** [emporia-vue-local/esphome#159] [Q&A]
I was wondering if it is possible to use other "generic" CTs with this monitor.  I'm assuming the ones that come with the kit are the xxxA/1v variety?  I would like to get 20A and 60A CTs that are a bit lower profile as there isn't much room in my panel.

I didn't see any configuration for the CTs in the esp, so I'm assuming they are hard coded in firmware or hardware somewhere else?  Is there a way to just scale the values of I'm using a different rating CT?  Say it expects a 50A and I'm using 100A, I would double the current reading it thinks it sees.

**Comment by ADKMechETECH** [comment]
My bad... Just found the other discussion.  Doesn't look too promising.

Anyone have any updated info?

**Reply by flaviut** [reply]
I know at least a couple people have had success, including here: https://github.com/emporia-vue-local/esphome/discussions/59#discussioncomment-4327213
